src/actionModifiers/ImageExport/ImageExportModifier.js

```diff
@@ -0,0 +1,37 @@
+import SessionService from 'services/session/SessionService';
+import { formatNumberSequence } from 'ommUtils/strings';
+
+function imageExportModifier(openmct) {
+  const PNGImageExportAction = openmct.actions._allActions['export-as-png'];
+  const JPGImageExportAction = openmct.actions._allActions['export-as-jpg'];
+  const imageExportActions = [PNGImageExportAction, JPGImageExportAction].filter(Boolean);
+  const sessionService = SessionService();
+
+  imageExportActions.forEach((action) => {
+    const invoke = action.invoke;
+
+    action.invoke = (objectPath, view) => {
+      let filename = objectPath[0].name;
+      const sessionFilter = sessionService.getHistoricalSessionFilter();
+
+      if (sessionFilter) {
+        filename = `${filename} - ${historicalFilterString(sessionFilter)}`;
+      }
+
+      filename = `${filename} - plot`;
```

Comment:
I think we should get the type from the `domainObject.type`, in case we use the image export from something else besides a plot

Reply:
wow. yeah, that's quite the oversight on my part, good catch.

Reply:
done did it